extensions/podman/packages/extension/src/extension.ts

```diff
@@ -1892,8 +1929,15 @@ export async function getJSONMachineListByProvider(containerMachineProvider?: st
 }
 
 export async function deactivate(): Promise<void> {
-  stopLoop = true;
   console.log('stopping podman extension');
+  // cleanup
+  stopLoop = true;
+  listeners.clear();
+  podmanServerVersions.clear();
+  podmanMachinesInfo.clear();
+  currentConnections.clear();
+  containerProviderConnections.clear();
+
```

Comment:
Is this related to this issue (except podmanServerVersions.clear())?

Reply:
missing a rebase